LibVLCSharp/Shared/LibVLC.cs

```diff
@@ -403,6 +409,10 @@ void UnsetLog()
                     {
                         UnsetLog();
                     }
+                    else
```

Comment:
Why do you need to reset logs?

Reply:
Since the `EventHandler<LogEventArgs>` is given to the native side, it has to be updated on every sub/unsub otherwise when invoked it may not be the last version.

Reply:
Can we pass the LibVLC instance then? That way, it doesn't change.

Reply:
What do you mean? We need to retrieve the logger in a static context from native code to raise events. How do we that with only a `LibVLC` instance?

Reply:
Pass the libvlc instance (aka `this`) to the GCHandle.
Then, from the callback :
`libvlc._log...`